<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="adwaitharyan" failures="1" tests="3" name="Test" time="8.516" errors="0" timestamp="2025-07-18T15:28:42 IST">
  <testcase classname="Listeners.OrangeHRM" name="testLogo" time="0.042"/>
  <testcase classname="Listeners.OrangeHRM" name="testAppUrl" time="0.013">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at Listeners.OrangeHRM.testAppUrl(OrangeHRM.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="Listeners.OrangeHRM" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- Test -->
